### PR TITLE
Annotate SIG network testfail emails

### PR DESCRIPTION
**Pull request by cmluciano**

### CODE REVIEW

File: testgrid/config/config.yaml [hunk 1]
```diff
@@ -223,6 +223,8 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-sig-cli
 - name: ci-kubernetes-e2e-gci-gce-ingress
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-ingress
+  alert_stale_results_hours: 24
+  num_failures_to_alert: 3
```
Comment (MrHohn) [inline testgrid/config/config.yaml]: Seems like this is the same as default, do we need to redefine it?
https://github.com/kubernetes/test-infra/blob/e75dd6e6fed04c554b5f254f4198f944bf6ded49/testgrid/config/config.yaml#L24

File: testgrid/config/config.yaml [hunk 2]
```diff
@@ -474,6 +478,8 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-alpha-features-release-1-6
 - name: ci-kubernetes-e2e-gci-gce-ingress-release-1-6
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-ingress-release-1-6
+  alert_stale_results_hours: 24
+  num_failures_to_alert: 5
```
Comment (MrHohn) [inline testgrid/config/config.yaml]: Why is ingress CI different (using 5 instead of 3)?

**Comment by krzyzacy** [comment]
/approve
I'll let @MrHohn stamp

File: testgrid/config/config.yaml (hunk 1 above)
Comment (cmluciano) [reply testgrid/config/config.yaml]: sry didn't realize there were defaults, I will update

File: testgrid/config/config.yaml (hunk 2 above)
Comment (cmluciano) [reply testgrid/config/config.yaml]: typo, will revert to default

**Comment by cmluciano** [comment]
comments addressed @MrHohn

**Comment by MrHohn** [comment]
Thanks!

/lgtm

**Pull request merged**